src/entry.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/sh
 set -e
 
-mkdir /dev/sgx
-ln -s /dev/sgx_enclave /dev/sgx/enclave
+if [ ! -d /dev/sgx ]; then
+	mkdir -p /dev/sgx
+	ln -s /dev/sgx_enclave /dev/sgx/enclave
+fi
```

Comment:
I guess it can't hurt to adopt this. I suggest changing it such that it also works for the case that the directory exists, but doesn't contain the link.
```suggestion
mkdir -p /dev/sgx
if [ ! -L /dev/sgx/enclave ]; then
	ln -s /dev/sgx_enclave /dev/sgx/enclave
fi
```

Reply:
Yep, sounds good.